### PR TITLE
url fix for slug and missing django template ending brackets

### PR DESCRIPTION
**Pull request by staranjeet**
For the profile url, there should be a `-` in the regex for url. 
Also it fixed the missing closing template tags in `profile/index.html`

### CODE REVIEW

File: wye/urls.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
         include('wye.organisations.urls', namespace="organisations")),
     url(r'^workshop/',
         include('wye.workshops.urls', namespace="workshops")),
-    url(r'^profile/(?P<slug>[a-zA-Z0-9]+)/$',
+    url(r'^profile/(?P<slug>[a-zA-Z0-9-]+)/$',
```
Comment (vnbang2003) [inline wye/urls.py]: why -  ?

Comment (staranjeet) [inline wye/urls.py]: slug is only the username then?

Comment (vnbang2003) [inline wye/urls.py]: slug can be anything with alphanumeric.

Comment (staranjeet) [inline wye/urls.py]: ohh I thought it contains `-` as well. Ok I will close this PR

**Pull request closed without merging**